### ISSUE TITLE
Add Voronoi regions

### ISSUE THREAD

**Issue by hiulit** [hiulit/Godot-3-2D-Destructible-Objects#1]
To have a better scattering.

![D-zZ1v8XUAAg_Gf](https://user-images.githubusercontent.com/481475/61219116-ca31f680-a713-11e9-9b7b-28a100b093d4.png)

We will need a way to create sprites and its respective collisions following this pattern.

**Comment by Xrayez** [comment]
I think this could be achieved via Delaunay triangulation which should be available in 3.2. 😉
godotengine/godot#29127.

[Delaunay_triangulation relationship with the Voronoi diagram](https://en.wikipedia.org/wiki/Delaunay_triangulation#Relationship_with_the_Voronoi_diagram).

**Comment by hiulit** [comment]
That looks amazing! Thanks! We'll wait for 3.2 then ;)
I would still need help though, hehe!

Btw, I've just realized that while researching how to create polygons out from sprites I came across this project that it turns out it's yours! https://github.com/Xrayez/godot-mesh-instance-collision-2d

**Comment by Xrayez** [comment]
Yeah, I got notified when you starred my project and then I've noticed this. 🙂

I have my own ecosystem of destructible stuff so I'm not sure whether I'd contribute, but always nice to see destructible stuff in Godot, perhaps this would be useful for some destructible parts that should truly break into pieces (like glass), so we'll see.
